adoc/bp_chap_expanded_support.adoc

```diff
@@ -419,6 +419,8 @@ Some actions are not yet supported:
 * Patch and errata support
 * Bare metal installations, PXE booting, and virtual host provisioning
 * Live patching
+* If you use `server.susemanager.fromdir = ...` option in rhn.conf of SUSE Manager, Ubuntu Client Tools will not work.
+  (reason: SMT cannot mirror Debian repositories (yet) and so cannot create it in that directory the correct files. We are waiting for SMT to release the feature/fix to mirror Debian repositories). 
```

Comment:
This is the reason, but I am not sure if we should add the reason to the docs. And RMT has the same problem.

Reply:
Agreed. We can make the reason as a comment as a remember to remove it whenever SMT/RMT fixes it.

Reply:
L423: there's something missing from this sentence. Also, the parentheses are nasty. What about something like this:

``This occurs because SMT cannot currently mirror Debian repositories, so it cannot create the required files in the directory. This will be addressed in a future release.``